Python/Filters/tdms_to_h5.py

```diff
@@ -0,0 +1,285 @@
+    else:
+      h5_group.attrs[key] = value
+
+class TDMStoH5(Filter):
+  def __init__(self) -> None:
+    self.area_offset: int = 0
+    self.intensity_offset: int = 0
+    self.laser_offset: int = 0
+    self.output_folder: str = ''
+    self.group: str = ''
+    self.stack_info_param: StackFileListInfo = StackFileListInfo(0, 0, 0, 0, 1, '', 'Prefix_', '_Suffix', 'tdms')
+
+  def _set_area_offset(self, value: int) -> None:
+    self.area_offset = value
+
+  def _get_area_offset(self) -> int:
+    return self.area_offset
+
+  def _set_intensity_offset(self, value: int) -> None:
+    self.intensity_offset = value
+
+  def _get_intensity_offset(self) -> int:
+    return self.intensity_offset
+
+  def _set_laser_offset(self, value: int) -> None:
+    self.laser_offset = value
+
+  def _get_laser_offset(self) -> int:
+    return self.laser_offset
+
+  def _set_output_folder(self, value: str) -> None:
+    self.output_folder = value
+
+  def _get_output_folder(self) -> str:
+    return self.output_folder
+
+  def _set_group(self, value: str) -> None:
+    self.group = value
+
+  def _get_group(self) -> str:
+    return self.group
+  
+  def _get_stack_info_param(self) -> StackFileListInfo:
+    return self.stack_info_param
+
+  def _set_stack_info_param(self, value) -> None:
+    self.stack_info_param = value
+
+  @staticmethod
+  def name() -> str:
+    return 'TDMS to H5'
+
+  @staticmethod
+  def uuid() -> str:
+    return '{8b069c55-6d94-4db1-b012-cdfb7dd08ad6}'
+
+  @staticmethod
+  def group_name() -> str:
+    return 'Example'
+
+  @staticmethod
+  def sub_group_name() -> str:
+    return 'Sub Example'
+
+  @staticmethod
+  def human_label() -> str:
+    return 'Convert TDMS to HDF5'
+
+  @staticmethod
+  def version() -> str:
+    return '1.0.0'
+
+  @staticmethod
+  def compiled_lib_name() -> str:
+    return 'Python'
+
+  def setup_parameters(self) -> List[FilterParameter]:
+    req = MultiDataArraySelectionFilterParameter.RequirementType([IGeometry.Type.Image], [AttributeMatrix.Type.Cell], [], [])
+    return [
+      IntFilterParameter('Area Offset', 'area_offset', self.area_offset, FilterParameter.Category.Parameter, self._set_area_offset, self._get_area_offset, -1),
+      IntFilterParameter('Intensity Offset', 'intensity_offset', self.intensity_offset, FilterParameter.Category.Parameter, self._set_intensity_offset, self._get_intensity_offset, -1),
+      IntFilterParameter('Laser Offset', 'laser_offset', self.laser_offset, FilterParameter.Category.Parameter, self._set_laser_offset, self._get_laser_offset, -1),
+      OutputPathFilterParameter('Output Folder', 'Output Folder', '', FilterParameter.Category.Parameter,
+                               self._set_output_folder, self._get_output_folder, -1),
+      StringFilterParameter('Group', 'group', self.group, FilterParameter.Category.Parameter, self._set_group,
+                           self._get_group, -1)

```

Comment:
There needs to be a comma at the end of this line otherwise I get a syntax error.